.github/workflows/ci.yml

```diff
@@ -252,6 +252,39 @@ jobs:
           path: ${{ github.workspace }}/.ccache
           key: ccache_all_windows_${{ github.sha }}
 
+  build_test_all_macos:
+    needs: setup
+    if: needs.setup.outputs.should-run == 'true' && needs.setup.outputs.ci-stage == 'postsubmit'
+    runs-on:
+      - ${{ github.repository == 'openxla/iree' && 'self-hosted' || 'macos-11' }}  # must come first
+      - runner-group=postsubmit
+      - os-family=macOS
+    env:
+      BUILD_DIR: build-macos
+    steps:
+      - name: "Checking out repository"
+        uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
+      - name: "Updating git submodules"
+        run: git submodule update --init --jobs 8 --depth 1
```

Comment:
Does this explicit submodule fetch outperform 

```yaml
    steps:
      - name: "Checking out repository"
        uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
        with:
          submodules: true
```
? (On Windows we saw poor behavior in the checkout action and wanted to ensure we were doing just what we wanted)

Reply:
Good question. I just copied from what you have in the Windows session there. :) It takes 2s to checkout the repo and 34s to fetch submodules this way. Let me try it out the above.

Reply:
Using the above takes 1min30s: https://github.com/openxla/iree/actions/runs/4327186867/jobs/7555521072. So yeah explicit submodule checkout saves ~1min.